### DISCUSSION TITLE
Request to add support for jem branch

### DISCUSSION THREAD

**Discussion by c0pt4n** [ROM-builders/temporary#20501] [Ideas]
Please sir add support for jem branch of evox 

https://github.com/Evolution-X/manifest/tree/tiramisu-jem

**Comment by Apon77** [comment]
why is this needed?

**Reply by c0pt4n** [reply]
> why is this needed?

My device needs jemalloc allocator

**Reply by Apon77** [reply]
I don't think they created a branch only for a device only.

**Reply by tedomi2705** [reply]
many devices need jemalloc allocator for miui camera

**Reply by Apon77** [reply]
cant occupy 300GB on server because of only a camera and only one rom having a different branch to do so.

**Reply by tedomi2705** [reply]
> cant occupy 300GB on server because of only a camera and only one rom having a different branch to do so.

it can just use existing tiramisu directory without creating separate tiramisu-jem directory
